editor/plugins/navigation_mesh_editor_plugin.h

```diff
@@ -31,6 +31,8 @@
 #ifndef NAVIGATION_MESH_GENERATOR_PLUGIN_H
 #define NAVIGATION_MESH_GENERATOR_PLUGIN_H
 
+#define RECAST_ENABLED
```

Comment:
This is wrong, `RECAST_ENABLED` is already defined if recast is compiled (in `modules/recast/SCsub`). If it's not compiled, forcing it to be defined 1) defeats the purpose of the compile define. 2) will likely prevent building as it links to files which are not compiled.